app/graphql/types/query_type.rb

```diff
@@ -59,6 +59,16 @@
     resolve ->(obj, args, ctx) { User.find_by(slug: args["slug"])}
   end
 
+  field :userByFirstLast do
```

Comment:
FirstLast is not very clear. Either userByName or userByFirstLastName makes more sense imo